src/main/java/arrays/Arrays.java

```diff
@@ -0,0 +1,85 @@
+package main.java.arrays;
+
+public class Arrays {
+    public static void main(String[] args) {
+
+        boolean[] myBoolean = new boolean[3];
+        byte[] myByte = new byte[4];
+        short[] myShort = new short[5];
+        char[] myChar = new char[6];
+        int[] myNum = new int[7];
+        float[] myFloat = new float[8];
+        long[] myLong = new long[2];
+        double[] myDoub = new double[1];
+        String[] myString = new String[10];
+
+        myChar[0] = 'a';
+        myChar[1] = 'b';
+        myChar[2] = 'c';
+        myChar[3] = 'd';
+        myChar[4] = 'e';
+        myChar[5] = 'f';
+
+        myString[0] = "sunday";
+        myString[1] = "monday";
+        myString[2] = "tuesday";
+        myString[3] = "wednesday";
+        myString[4] = "thursday";
+        myString[5] = "friday";
+        myString[6] = "saturday";
+        myString[7] = "grapefruit";
+        myString[8] = "grape";
+        myString[9] = "kiwi";
+
+
+        for(int r = 0; r < myBoolean.length; r++) {
```

Comment:
what was the reason to use `r` as a name? correct name in for loop is `i` or `j` (if it's internal loop of another loop). 

Check this https://www.w3schools.com/java/java_scope.asp